android/build.gradle

```diff
@@ -1,6 +1,30 @@
 android {
     buildToolsVersion "28.0.3"
     compileSdkVersion 27
+
+    signingConfigs {
+        debugConfig
+                {
+                    storeFile file("./debugKeystore.ks")
```

Comment:
please remove this. I am aware that you need a special keystore for Huawei IAP to work, but this way the example project won't work any more if there is no keystore file present.